conversations.yaml

```diff
@@ -374,6 +384,30 @@ components:
             notification_type:
               type: string
               enum: [REGULAR, CUSTOM]
+    EndOfConversationSignal:
+      properties:
+        recipient:
+          properties:
+            id:
+              type: string
+              description:
+                An identifier unique to the customer, such as phone number,
+                email, or another channel-specific value. For phone numbers,
+                the identifier must be in E.164 format.
+        sender_action:
+          type: string
+          description: The value is always 'conversation_ended'.
+        reason:
+          type: string
+          description:
+            "The reason the conversation has ended. \
+            'complete': The conversation \ reached the end.\
+             The signal is sent after the last dialog message. \
+            'optout': The respondent opted-out during the conversation. \
+            'expired': Sent after attempt to interact with an expired conversation.\
+            'sampling': The respondent excluded from survey due to Quarantine filter in sampling.\
+            'failed': When an error has occured"
```

Comment:
Remove the enum items from the description.  Just leave it as `The reason the conversation has ended.`